trafilatura/xml.py

```diff
@@ -269,34 +272,50 @@ def merge_with_parent(element, include_formatting=False):
     parent.remove(element)
 
 
+def process_element(element, returnlist, include_formatting):
+    # Process children recursively
+    if element.text is not None:
+        # this is the text that comes before the first child
+        textelement = replace_element_text(element, include_formatting)
+        returnlist.append(textelement)
+
+    for child in element:
+        if child is not None:
```

Comment:
By definition, `list(element)` in LXML should return existing elements, is there a problem if we remove this line?